doc/release/1.10.2-notes.rst

```diff
@@ -0,0 +1,62 @@
+NumPy 1.10.2 Release Notes
+**************************
+
+This release deals with a number of bugs that turned up in 1.10.1 and
+adds various build and release improvements.
+
+Numpy 1.10.1 supports Python 2.6 - 2.7 and 3.2 - 3.5.
+
+Changes
+=======
+
+Testing with development Numpy versions will raise exceptions for
+``RuntimeWarning`` and ``DeprecationWarning``, released Numpy versions will
+leave them as warnings. Previously, development branches other than
+``master`` would not raise exceptions for these warnings.
```

Comment:
I find this paragraph hard to decipher.
Does it mean previously maintenance branches did not raise but now they do?
if so I don't think that this needs to be so prominent in the release notes, its not important for upgrades.